android/fakedapp/src/main/java/com/solana/mobilewalletadapter/fakedapp/MainViewModel.kt

```diff
@@ -310,10 +375,26 @@ class MainViewModel : ViewModel() {
                     Log.e(TAG, "Message payload invalid", cause)
                 is JsonRpc20Client.JsonRpc20RemoteException ->
                     when (cause.code) {
-                        ProtocolContract.ERROR_REAUTHORIZE -> Log.e(TAG, "Reauthorization required", cause)
-                        ProtocolContract.ERROR_AUTHORIZATION_FAILED -> Log.e(TAG, "Auth token invalid", cause)
-                        ProtocolContract.ERROR_NOT_SIGNED -> Log.e(TAG, "User did not authorize signing", cause)
-                        ProtocolContract.ERROR_TOO_MANY_PAYLOADS -> Log.e(TAG, "Too many payloads to sign", cause)
+                        ProtocolContract.ERROR_REAUTHORIZE -> Log.e(
+                            TAG,
+                            "Reauthorization required",
+                            cause
+                        )
+                        ProtocolContract.ERROR_AUTHORIZATION_FAILED -> Log.e(
+                            TAG,
+                            "Auth token invalid",
+                            cause
```

Comment:
Ugh, can we pleas not have our code auto format this way? It's so many newlines for very limited readablity.

Reply:
Ugh yeah 😬 I thought I had reverted these auto-format changes. Will try to revert them back from the source.

Reply:
Should be good now